modular_skyrat/modules/aesthetics/pda/pda.dm

```diff
@@ -0,0 +1,192 @@
+
+//greyscale config
+/datum/greyscale_config/pda/aesthetics
+	icon_file = 'modular_skyrat/modules/aesthetics/pda/pda.dmi'
+	json_config = 'code/datums/greyscale/json_configs/pda.json'
+
+/datum/greyscale_config/pda/aesthetics/chaplain
+	json_config = 'code/datums/greyscale/json_configs/pda_chaplain.json'
+/*Disabled as it's haven't been use yet
+/datum/greyscale_config/pda/aesthetics/clown
+	json_config = 'code/datums/greyscale/json_configs/pda_clown.json'
+*/
+/datum/greyscale_config/pda/aesthetics/head
+	json_config = 'code/datums/greyscale/json_configs/pda_head.json'
+
+/datum/greyscale_config/pda/aesthetics/mime
+	json_config = 'code/datums/greyscale/json_configs/pda_mime.json'
+
+/datum/greyscale_config/pda/aesthetics/stripe_split
+	json_config = 'code/datums/greyscale/json_configs/pda_stripe_split.json'
+
+/datum/greyscale_config/pda/aesthetics/stripe_thick
+	json_config = 'code/datums/greyscale/json_configs/pda_stripe_thick.json'
+
+/datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	json_config = 'code/datums/greyscale/json_configs/pda_stripe_thick_head.json'
+
+//PDA GAGS
+/obj/item/pda
+	greyscale_config = /datum/greyscale_config/pda/aesthetics
+	greyscale_colors = "#a2b4cf#6ab73b"
+
+/obj/item/pda/medical
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick
+	greyscale_colors = "#d6d9e0#262d99#65c9ff"
+
+/obj/item/pda/viro
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_split
+	greyscale_colors = "#d6d9e0#6ab73b#6df782"
+
+/obj/item/pda/engineering
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick
+	greyscale_colors = "#dab424#2a3198#a22626"
+
+/obj/item/pda/security
+	greyscale_colors = "#d13c3c#6ab73b"
+
+/obj/item/pda/detective
+	greyscale_colors = "#c66a42#262d99"
+
+/obj/item/pda/warden
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick
+	greyscale_colors = "#d13c3c#6ab73b#ffc900"
+
+/obj/item/pda/janitor
+	greyscale_colors = "#bda2cf#d13c3c"
+
+/obj/item/pda/toxins
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick
+	greyscale_colors = "#d6d9e0#262d99#c28be5"
+
+/obj/item/pda/heads
+	greyscale_config = /datum/greyscale_config/pda/head
+	greyscale_colors = "#a2b4cf#262d99"
+
+/obj/item/pda/heads/hop
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#a2b4cf#262d99#65c9ff"
+
+/obj/item/pda/heads/hos
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#d13c3c#262d99#65c9ff"
+
+/obj/item/pda/heads/ce
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#dab424#262d99#65c9ff"
+
+/obj/item/pda/heads/cmo
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#d6d9e0#262d99#65c9ff"
+
+/obj/item/pda/heads/rd
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#d6d9e#262d99#c28be5"
```

Comment:
```suggestion
	greyscale_colors = "#d6d9e0#262d99#c28be5"
```
You forgot the 0 here and I suspect that's why you're failing integration tests.